stellarphot/core.py

```diff
@@ -541,7 +546,7 @@ def __init__(
 
             # Apply the filter/passband name update
             if passband_map is not None:
-                self._passband_map = passband_map.copy()
+                self._passband_map = passband_map.model_copy()
```

Comment:
if `passband_map` is the `dict` passed in as a parameter, how can we apply `model_copy()` which I though was from pydantic?  Since this file is not importing the `PassbandMap` model from `.settings/models.py` I am having a hard time seeing how that model gets applied.